client/src/pages/HomePage/index.js

```diff
@@ -1,23 +1,49 @@
-import React, { useState } from "react";
-import { Button } from "../../components";
-import { MomentumClient } from "../../common";
+import React from "react";
+import { Flex, Text, Button, Image } from "@chakra-ui/core";
+import { Link } from "react-router-dom";
+
+const HomePageButton = ({ text, to }) => {
```

Comment:
Just to explicitly clarify, are we using functional components or classes? We should keep it consistent throughout the app. Happy with either 😄

Reply:
Let's use functional components please 😺

Reply:
(this is using functional components if that's what you were asking)

Reply:
Yepyep, I just wanted to confirm that it was intentional so we don't have to refactor later!